python/ray/services.py

```diff
@@ -816,7 +816,7 @@ def start_ui(redis_address, stdout_file=None, stderr_file=None, cleanup=True):
     # We generate the token used for authentication ourselves to avoid
     # querying the jupyter server.
     new_notebook_directory, webui_url, token = (
-        get_ipython_notebook_path(port))
+        get_ipython_notebook_path(get_node_ip_address(), port))
```

Comment:
Thanks for contributing this. I think that on AWS this will return the node's "private IP address", when we actually need the "public IP address". Do you know if that's the case?

Reply:
We didn't deploy Ray Cluster In AWS, but in our machines which just support access by ssh.
So we needed to access the web UI in our personal computer.

In same of our machines, the external network is open, so we can access the web UI directly.
In other machines, the external network is close, we can access the web UI by proxy.

So, any suggest for this？